lib/Doctrine/DBAL/Types/JsonArrayType.php

```diff
@@ -70,12 +51,5 @@ public function getName()
     {
         return Type::JSON_ARRAY;
     }
-
-    /**
-     * {@inheritdoc}
-     */
-    public function requiresSQLCommentHint(AbstractPlatform $platform)
```

Comment:
So basically, until 3.0, we should keep the new one always commented, and then switch it out when the old one will be dropped ?